rnd/autogpt_server/autogpt_server/util/service.py

```diff
@@ -43,7 +46,9 @@ def run(self):
 
 class AppService(AppProcess):
     shared_event_loop: asyncio.AbstractEventLoop
+    event_queue: AsyncEventQueue = AsyncRedisEventQueue()
```

Comment:
Just a note that this is generally a bad pattern as it will cause the `event_queue` instance to be shared across all `AppService` instances. In this specific case it probably won't cause any problems because every `AppService` runs in a separate process.